client/cmd/auth.go

```diff
@@ -241,13 +242,16 @@ func Cancel(username string, password string, yes bool) error {
 	}
 
 	err = auth.Delete(c, username)
-
-	if err != nil {
+	cleanup := fmt.Errorf("\n%s %s\n\n", "409", "Conflict")
+	if reflect.DeepEqual(err, cleanup) {
```

Comment:
I can file an issue, yeah. Given how the error is raised way down in the `http.go` portion then it would be a bit of a change required to pull that off since it wouldn't be a one off.